Create first capstone project africa ai expo

Lembani opened the pull request:
### This PR is for the first capstone project. The project name is Africa AI Expo.

[Here is a link to the project documentation video](https://www.loom.com/share/3a23a8af2b5244da93529458d7855e26)

This project has the following for both Mobile and Desktop:
1. Home page
- Header section
- Main program section
- Speaker section (Dynamically rendered)
- Footer section

2. About page
- Header section
- Logo section
- Past Expo's section
- Footer section

Lembani committed:
Add Logo image

Lembani committed:
Add main program images

Lembani committed:
Add navbar styles for mobile

Lembani committed:
Add headline style for mobile

Lembani committed:
Update mobile description styles

Lembani committed:
Add main program section for mobile

Lembani committed:
Add navbar markup for mobile and desktop

Lembani committed:
Add headline section makrup:
- mobile
- desktop

Lembani committed:
Add main prgram section

Lembani committed:
Add navbar markup on about page
- mobile
- desktop

Lembani committed:
Add headline section
- mobile desktop

Lembani committed:
Add main program section for mobile

Lembani committed:
Add Mobile styles for all sections on:
- Home page
- About page

Lembani committed:
Update styles to fix linter errors

Lembani committed:
Update home page with all sections
- Mobile menu
- Speakers section
- Footer section
- etc

Lembani committed:
Add About page with all sections
- Header section
- Mobile menu
- program section
- Footer section
- etc

Lembani committed:
Add images for all sections and backgrounds

Lembani committed:
Update about markup to fix linter errors

Lembani committed:
Create Script:
- to enable mobile menu interaction
- dynamic year updating in footer

Lembani committed:
Update to fix linter errors

Lembani committed:
Update Syles for desktop on Home & About pages
- Header section
- Program section
- Speakers section
- Sponsors section
- Footer section
- etc

Lembani committed:
Add Script to render speakers dynamically

Cathella commented on index.html:
```diff
@@ -0,0 +1,136 @@
+        <h2 class="program-heading">Main Program</h2>
+        <hr class="program-line">
+        <div class="program-container">
+            <div class="program-item">
+                <img src="./images/program1.png" alt="Lecture" class="program-icon">
+                <h4 class="title">Speak</h4>
+                <p class="program-description">Lorem ipsum dolor sit amet consectetur.</p>
+            </div>
```
- [x] To make the code more semantic, I would suggest making the `div` wrappers with the program-item class changed to `article` tags

```
<article class="program-item"></article>
```

Lembani replied:
Hi @Cathella, Thanks for the review. 
I have fixed the issues above. Kindly review them before I submit another request for review.
Regards.

Cathella reviewed (changes requested):
## 🟡 Status: Changes Required 🛑

### Hi @Lembani 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Descriptive PR and README.
✅ Used GitHub Pages to deploy the project.
✅ Project well styled as seen in Behance design


## Required Changes ♻️
_Check the comments under the review._

- [x] When I hover over the link in the mobile nav, the link becomes invisible, this is because the onHover link color is the same as the background color of the nav. Kindly change the link color.

![image](https://user-images.githubusercontent.com/4247740/163685828-818a52f7-432c-4b0f-afc5-61f3d12ee04f.png)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Lembani committed:
Update to fix:
- hover effect on mibile menu
- Make program section more semantic by using _article_ tag

Lembani committed:
Update to fix typo on article

phelian23 reviewed (approved):
Hi @Lembani, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Cheers and Happy coding!👏👏👏

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
